x-pack/plugins/observability/public/pages/slos/components/slo_list_item.tsx

```diff
@@ -141,13 +139,7 @@ export function SloListItem({
               <EuiFlexGroup direction="column" gutterSize="m">
                 <EuiFlexItem>
                   <EuiText size="s">
-                    {slo.summary ? (
-                      <EuiLink data-test-subj="o11ySloListItemLink" onClick={handleViewDetails}>
-                        {slo.name}
-                      </EuiLink>
-                    ) : (
-                      <span>{slo.name}</span>
-                    )}
+                    {slo.summary ? <a href={sloDetailsUrl}>{slo.name}</a> : <span>{slo.name}</span>}
```

Comment:
Should we keep `data-test-subj="o11ySloListItemLink"`?

Reply:
I'll keep it in case it's used by fullstory you're right 👍🏻